src/main/java/slimeknights/tconstruct/library/recipe/tinkerstation/building/ToolBuildingRecipeBuilder.java

```diff
@@ -39,6 +40,7 @@ public void save(Consumer<FinishedRecipe> consumerIn) {
   @Override
   public void save(Consumer<FinishedRecipe> consumerIn, ResourceLocation id) {
     ResourceLocation advancementId = this.buildOptionalAdvancement(id, "parts");
-    consumerIn.accept(new LoadableFinishedRecipe<>(new ToolBuildingRecipe(id, group, output, outputSize, extraRequirements), ToolBuildingRecipe.LOADER, advancementId));
+    slotId = id;
```

Comment:
This is the wrong way to do this, the slot ID will not be the recipe ID. Just add a setter for the slot ID.